docs/src/example.md

```diff
@@ -34,13 +34,13 @@ Let's define a new capacity factor as 0.6
 target_cp = 0.6
 ```
 
-We can use the function ``fit\_profile`` in this package to obtain the coefficient that fit the values to the target.
+We can use the function ``find\_solution`` in this package to obtain the coefficient that fit the values to the target.
```

Comment:
```suggestion
We can use the function `find_solution` in this package to obtain the coefficient that fit the values to the target.
```
The single backticks transform into code, the double backticks transform into math.